### ISSUE TITLE
Laravel Mix Support

### ISSUE THREAD

**Issue by erayerdin** [erayerdin/sos-django-template#50]
This issue, if closed, closes #7.

Laravel Mix is a great solution that solves %80 of the use case.

SPA is not needed for many projects. VueJS is a good solution for integrating AJAX to some parts of web page, e.g. a search bar or a list with pagination.

# To Do

 - [ ] `createapp` command to start an app with Laravel Mix with its gitignore
 - [ ] `mix` command for compiling mix assets
 - [ ] `serve` command to watch mix assets and run Django server in parallel

**Comment by erayerdin** [comment]
React is widely adopted and Laravel Mix support is not needed anymore.
